docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -991,6 +991,16 @@ The FormControlLabel's `componentsProps` prop was deprecated in favor of `slotPr
  />
 ```
 
+## ListItem
+
+### ContainerComponent
+
+The ListItem's `ContainerComponent` prop was deprecated in favor of `slots.root` or `component` instead.
```

Comment:
`slots.root` and `component` are not a direct replacement for `ContainerComponent`, since the latter was only rendered if the consumers was using `ListItemSecondaryAction`, but not sure it's worth going into more detail here.